### DISCUSSION TITLE
Migrate from xwiki to wiki.js

### DISCUSSION THREAD

**Discussion by codesmaker** [requarks/wiki#5460] [Help / Questions]
Hi,

First of all, thanks a lot for the great wiki.

I'm currently using xwiki [https://www.xwiki.org/xwiki/bin/view/Main/WebHome](url). Is it possible to migrate my articles there to wiki.js?

Thanks

**Comment by codesmaker** [comment]
Any ideas?

**Reply by NGPixel** [reply]
As long as you can export your content to either Markdown or HTML files, you can easily import them using the Local Disk storage module.
